.pylintrc

```diff
@@ -0,0 +1,51 @@
+[MASTER]
+
+# Add files or directories to the blacklist.
+ignore=LICENSE,
+       MANIFEST.in,
+       PUBLISH.md,
+       README.md,
+       requirements.txt,
+       script.test,
+
+
+[MESSAGES CONTROL]
+
+# Types of check we want to disable
+# format
+# disable=stuff-that-we-actually-want-ignored,
+#        stuff-that-we-actually-want-ignored2,
+#
+#        stuff-thats-only-ignored-for-now-and-will-be-addressed-in-a-pr,
+#        stuff-thats-only-ignored-for-now-and-will-be-addressed-in-a-pr2
+disable=import-error,
+        no-name-in-module,
```

Comment:
I wouldn't classify those checks as wanted to be ignored.

Reply:
What all checks should be ignored then?

Reply:
@Rechi